### PR TITLE
[API][PAP] Add address stats

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Normalizer/Pap/AppendCampaignStatsNormalizer.php [hunk 1]
```diff
@@ -43,6 +47,8 @@ public function normalize($object, $format = null, array $context = [])
 
         $campaign['nb_surveys'] = $object->getCampaignHistoriesWithDataSurvey()->count();
         $campaign['nb_visited_doors'] = $this->campaignHistoryRepository->countVisitedDoors($object);
+        $campaign['nb_addresses'] = $object->getNbAddresses();
```
Comment (ottaviano) [inline src/Normalizer/Pap/AppendCampaignStatsNormalizer.php:50]: we can use `Group("pap_campaign_read")` on getter for do the same job

File: src/Normalizer/Pap/AppendCampaignStatsNormalizer.php [hunk 2]
```diff
@@ -43,6 +47,8 @@ public function normalize($object, $format = null, array $context = [])
...
+        $campaign['nb_voters'] = $object->getNbVoters();
```
Comment (elvino29) [inline src/Normalizer/Pap/AppendCampaignStatsNormalizer.php:51]: you can add the serialization group directly in the Campaign entity and remove those two line in the normalizer

**Review by ottaviano — APPROVED** [review]

File: src/Normalizer/Pap/AppendCampaignStatsNormalizer.php (hunk 1 above)
Comment (illusionOfParadise) [reply src/Normalizer/Pap/AppendCampaignStatsNormalizer.php:50]: We need to return this info only for Jemengage Admin, but using a group `pap_campaign_read` will return this info also for app mobile.

File: src/Repository/Pap/AddressRepository.php [hunk 3]
```diff
@@ -76,4 +77,22 @@ public function findNear(float $latitude, float $longitude, int $zoom, int $limi
 
         return $qb->getQuery()->getResult();
     }
+
+    public function countByPapCampaign(Campaign $campaign): int
```
Comment (elvino29) [inline src/Repository/Pap/AddressRepository.php:81]: parameter seems unused

File: src/Repository/Pap/AddressRepository.php [hunk 4]
```diff
@@ -76,4 +77,22 @@ public function findNear(float $latitude, float $longitude, int $zoom, int $limi
...
+    {
+        return (int) $this->createQueryBuilder('address')
+            ->select('COUNT(1)')
+            ->getQuery()
+            ->getSingleScalarResult()
+        ;
+    }
+
+    public function countVotersByPapCampaign(Campaign $campaign): int
```
Comment (elvino29) [inline src/Repository/Pap/AddressRepository.php:90]: same here

**Pull request merged**